collections/_development_standards/patterns.md

```diff
@@ -169,3 +169,49 @@ Another positive side-effects of using CQRS is that there are naturally less mer
 
 * [CQRS - Microsoft](https://docs.microsoft.com/en-us/azure/architecture/patterns/cqrs)
 * [CQRS - Martin Fowler](https://martinfowler.com/bliki/CQRS.html)
+
+
+### Managed Identities (MI)
+
+Using a managed identity allows an application to authenticate to any service that supports Azure AD authentication without having credentials.
```

Comment:
I think there needs to be a more of an explanation as to why we want the devs to use MI. If you take a look at the other sections on this page they try to give enough background on the technology benefits to (hopefully) remove any knee-jerk negative reactions without having too much that we are just replicating the official docs.

Reply:
Expanded section